Sources/Display.php

```diff
@@ -884,15 +885,15 @@ function Display()
 	{
 		$firstIndex = 0;
```

Comment:
By the way, this line doesn't need to be here. If it is not set on [line 1215](https://github.com/SimpleMachines/SMF2.1/pull/4522/files#diff-eeb7dcd590ea5ad96bd502e7c294017bL1215), a default value of 0 will be used anyway.